massa-network-exports/src/settings.rs

```diff
@@ -52,6 +52,10 @@ pub struct NetworkSettings {
     pub max_in_connection_overflow: usize,
     /// Max operations per message in the network to avoid sending to big data packet.
     pub max_operations_per_message: u32,
+    /// Read limitation for a connection in bytes per seconds
+    pub max_bit_read: u32,
```

Comment:
I think we should use u64 if in the future we want to go beyond a couple gigabytes per second (eg. restricted fiber networks)